esmvaltool/recipes/recipe_flato13ipcc.yml

```diff
@@ -692,6 +722,70 @@ diagnostics:
         # Plot style
         styleset: CMIP5
 
+  # **********************************************************************
+  # Flato et al. (2013) - IPCC AR5, chap. 9
+  # similar to fig. 9.14
+  # **********************************************************************
+  # SST zonal mean and equatorial, multi model mean
+  # **********************************************************************
+
+  fig09-14:
+    description: IPCC AR5 Ch. 9, Fig. 9.14 (sst error, zonal mean and equatorial)
+    themes:
+      - phys
+    realms:
+      - ocean
+    variables:
+      tos_zm:
+        short_name: tos
+        preprocessor: zonal_mean
+        reference_dataset: HadISST
+        mip: Omon
+        field: T2Ms
+      tos_eq:
+        short_name: tos
+        preprocessor: equatorial
+        reference_dataset: HadISST
+        mip: Omon
+        field: T2Ms
+    additional_datasets:
+      - {dataset: HadISST, project: OBS, type: reanaly, version: 1, start_year: 1979, end_year: 1999, tier: 2}
+      - {dataset: ACCESS1-0, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: ACCESS1-3, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CanCM4, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CCSM4, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CESM1-FASTCHEM, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CESM1-WACCM, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CMCC-CESM, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CMCC-CM, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CMCC-CMS, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CNRM-CM5, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: CSIRO-Mk3-6-0, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      # - {dataset: EC-EARTH, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      - {dataset: FGOALS-g2, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      # - {dataset: GFDL-CM2p1, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
+      # - {dataset: GFDL-CM3, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1979, end_year: 1999}
```

Comment:
Why are all those datasets commented out?

Reply:
Those models need fixes that are not yet in ESMValCore. They will be once ESMValGroup/ESMValCore#179 is merged.